docker/script/entrypoint.sh

```diff
@@ -23,6 +23,13 @@ export \
 install_requirements() {
     # Install custom python package if requirements.txt is present
     if [[ -e "$AIRFLOW_HOME/$REQUIREMENTS_FILE" ]]; then
+      if ! grep -- "-c " "$AIRFLOW_HOME/$REQUIREMENTS_FILE"
+      then
+          if ! grep -- "--constraint " "$AIRFLOW_HOME/$REQUIREMENTS_FILE"
```

Comment:
Nit: You could also combine these with an OR (using an escaped pipe) something like: `grep 'foo\|bar' /path/to/file`).

But it's a minor improvement, feel free to make it or not, approving :+1: